configs/phi-1_5/setup/requirements.txt

```diff
@@ -1,8 +1,8 @@
---extra-index-url https://download.pytorch.org/whl/cu118
-torch==2.3.0+cu118
-torchvision==0.18.0+cu118
-torchaudio==2.3.0+cu118
-transformers==4.41.0
+--extra-index-url https://download.pytorch.org/whl/cu124
```

Comment:
We may also need to update the ACA image to version 2.4.0+cu124.

Reply:
Updated!